sdks/java/core/src/main/java/org/apache/beam/sdk/expansion/ExternalTransformRegistrar.java

```diff
@@ -31,4 +34,27 @@
 
   /** A mapping from URN to an {@link ExternalTransformBuilder} class. */
   Map<String, Class<? extends ExternalTransformBuilder<?, ?, ?>>> knownBuilders();
```

Comment:
There should be a default implementation for this that just returns an empty Map, that way new implementations will only have to override `knownBuilderInstances()`

Reply:
Done